builtin/logical/jwt/path_config.go

```diff
@@ -0,0 +1,89 @@
+package jwt
+
+import (
+	"fmt"
+	"strings"
+	"errors"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfig(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: `roles/(?P<name>\w+)/config`,
```

Comment:
This has been removed in the upcoming PR. I've merged the config in with the role creation.